http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -359,3 +340,29 @@ class ProxyService(
     "Connection"
   ).map(_.toLowerCase)
 }
+
+object ProxyService {
+  // From the outside, we are going to support TWO paths to access Jupyter:
+  // 1)   /notebooks/{googleProject}/{clusterName}/
+  // 2)   /proxy/{googleProject}/{clusterName}/jupyter/
+  // 3)   /notebooks/{googleProject}/{clusterName}/jupyter/
```

Comment:
I think we can remove case 3 here. `/notebooks/{googleProject}/{clusterName}/jupyter/` was never a supported proxy path. We previously only had `/notebooks/{googleProject}/{clusterName}/`, and we are trying to migrate to `/proxy/{googleProject}/{clusterName}/jupyter/`.

I think FireCloud is probably the only system on the legacy path at this point..

Reply:
I added 3 because somehow in automation tests, all proxied requests have this path